pkg/convert/go_test.go

```diff
@@ -127,10 +107,92 @@ func TestFromGo(t *testing.T) {
 			}
 		]
 	}
-	`)
+	`),
+			expected: model.Translation{
+				Language: language.English,
+				Original: false,
+				Messages: []model.Message{
+					{
+						ID:          "1",
+						Message:     "{translatedMessage1}",
+						Description: "description1",
+						Positions:   []string{"src/config.go:10"},
+						Status:      model.MessageStatusFuzzy,
+					},
+					{
+						ID:          "2",
+						Message:     "{translatedMessage2}",
+						Description: "description2",
+						Positions:   []string{"src/config.go:20"},
+						Status:      model.MessageStatusFuzzy,
+					},
+					{
+						ID:          "3",
+						Message:     "",
+						Description: "description3",
+						Status:      model.MessageStatusUntranslated,
+					},
+				},
+			},
+		},
+		{
+			name: "Message with special chars",
+			input: []byte(`
+	{
+		"language": "en",
+		"messages": [
+			{
+				"id": "1",
+				"meaning": "description1",
```

Comment:
Maybe you could supplement test with more cases?
Example:
Message: Order #{Id} has been canceled for {ClientName},